src/applications/caregivers/helpers.js

```diff
@@ -19,7 +20,97 @@ export const medicalCenterLabels = Object.keys(caregiverFacilities).reduce(
 );
 
 // Turns the facility list for each state into an array of strings
-export const medicalCentersByState = _.mapValues(
+const medicalCentersByState = _.mapValues(
   val => val.map(center => center.code),
   caregiverFacilities,
 );
+
+// transforms forData to match fullSchema structure for backend submission
+const submitTransform = (formConfig, form) => {
+  // checks for optional chapters using ssnOrTin
+  const hasSecondaryOne =
+    form.data.secondaryOneSsnOrTin === undefined ? null : 'secondaryOne';
```

Comment:
We can use [null coalescing operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) now, I think... so this becomes

```js
const hasSecondaryOne = form.data.secondaryOneSsnOrTin = null ?? 'secondaryOne';
```